C/plugins/south/dht11/dht11.cpp

```diff
@@ -0,0 +1,146 @@
+/*
+ * FogLAMP south service plugin
+ *
+ * Copyright (c) 2018 OSIsoft, LLC
+ *
+ * Released under the Apache 2.0 Licence
+ *
+ * Author: Amandeep Singh Arora
+ */
```

Comment:
Not very, but looks doxygen has this format for this. 

```
/** @file dht11.cpp
 *  @brief FogLAMP south service DHT11 plugin
 *
 *   ....what this file is for, if more details required then brief  ....
 *  @author Amandeep Singh Arora
 *  @version ..if any..
 *  @license  Released under the Apache 2.0 Licence or just Apache 2.0
 */
```

Reply:
Doxygen format has not been used in anywhere in code base. I searched for "@author" recursively.